Minor changes to shelley.cddl for ease of use

rooooooooob opened the pull request:
In order for ease of code generation these changes, which don't impact
the CBOR representation, were made. These certificate/multisig changes
are the same ones that were done to relay before, as otherwise there is
not enough information for code generation tooling in each group choice
variant to get proper names from, as comments are discarded.

amount changing to coin was done to improve code generated and to make
it more clear for readers.

uroboros commented:
CI is failing on this transient issue - https://github.com/input-output-hk/cardano-ledger-specs/issues/1638 - if you -re-trigger the tests (with an empty commit, say) you should be ok

JaredCorduan reviewed (approved):
thanks @rooooooooob , this seem good to me (happy to hear it helps the generation).

The test failure has nothing to do with this PR, it was an unrelated property test failure that is statistical in nature.  I've logged the failure in #1638. one easy way to re-run CI is to change the sha (with `git commit --amend`) and force push. sorry for the hassle.

JaredCorduan commented:
somehow I missed @uroboros 's comments, making mine redundant. :) I also notice that the commit isn't signed, so I'll go ahead and sign at and force push, two birds.

JaredCorduan commented:
oops, it's from a fork, I can't sign it

rooooooooob committed:
Minor changes to shelley.cddl for ease of use

In order for ease of code generation these changes, which don't impact
the CBOR representation, were made. These certificate/multisig changes
are the same ones that were done to relay before, as otherwise there is
not enough information for code generation tooling in each group choice
variant to get proper names from, as comments are discarded.

amount changing to coin was done to improve code generated and to make
it more clear for readers.

Signed-off-by: rooooooooob <raftias@gmail.com>

rooooooooob commented:
I didn't have permissions to push a branch directly to this repo so I used a fork. I re-did it with --signoff, will it work now?

JaredCorduan commented:
> I didn't have permissions to push a branch directly to this repo so I used a fork. I re-did it with --signoff, will it work now?

it's still not show up on github as verified. you could try `git commit --amend -S` and then force pushing. but I am also happy to put up a PR myself and grab #1648 while I'm at it

JaredCorduan commented:
@rooooooooob you can close this PR if you are happy with #1660

rooooooooob commented:
Closing, I had tried --amend -s before. I think it was using a cached build that was failing which wasn't signed off since the code was the same and the failure pointed to the first one it failed on. But I'll just close this since that other PR covers the changes.
